### PR TITLE
Fish 6980 add ear libs to openapi scan ent

### PR DESCRIPTION
**Pull request by aubi**
Fix `mp.openapi.extensions.scan.lib` property name
Update scanning of jars in ear's `lib/` directory, change introduced in FISH-6980

Merge after finishing Community change: https://github.com/payara/Payara-Documentation/pull/238

### CODE REVIEW

**Comment by JamesHillyard** [comment]
FISH-6980 isn't going into the current release. Do not merge until documentation cuts for the release have been taken

File: community/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ In addition to the standard properties, the following properties have been added
 
 |===
 | Property key | Description
-| `mp.openapi.scan.lib`  |  Configuration property that allows the server to scan packaged JAR files inside the WAR's library directory (`WEB-INF/lib`). Default value is `false`.
+| `mp.openapi.extensions.scan.lib`  |  Configuration property that allows the server to scan packaged JAR files inside the WAR's library directory (`WEB-INF/lib`). If the WAR is inside EAR, the server scans also JAR files inside EAR's `lib/` directory. Default value is `false`.
```
Comment (Pandrex247) [inline community/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc:86]: Wording
```suggestion
| `mp.openapi.extensions.scan.lib`  |  Configuration property that allows the server to scan packaged JAR files inside the WAR's library directory (`WEB-INF/lib`). If the WAR is inside EAR, the server also scans JAR files inside the EAR's `lib/` directory. Default value is `false`.
```

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc [hunk 2]
```diff
@@ -64,7 +64,7 @@ Operations that want to specify an alternative list of servers must define an `o
 | `mp.openapi.schema.` a| Prefix of the configuration property to specify an alternative list of servers to service an operation.
 Prefix of the configuration property to specify a schema for a specific class, in JSON format. The remainder of the property key must be the fully-qualified class name. The value must be a valid OpenAPI schema object, specified in the JSON format. The use of this property is functionally equivalent to the use of the `@Schema` annotation on a Java class, but may be used in cases where the application developer does not have access to the source code of a class.
 
-When a `name` key is provided with a string value, the schema will be added to the `schemas` collection in the `components` object of the resulting OpenAPI document using ``name``'s value as the key.
+When a `name` key is provided with a string value, the schema will be added to the `schemas` collection in the `components` object of the resulting OpenAPI document using `name`'s value as the key.
```
Comment (Pandrex247) [inline enterprise/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc:67]: I don't think this renders correctly.
Looking at the preview it's wrong.

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc [hunk 3]
```diff
@@ -83,7 +83,7 @@ In addition to the standard properties, the following properties have been added
 
 |===
 | Property key | Description
-| `mp.openapi.scan.lib`  |  Configuration property that allows the server to scan packaged JAR files inside the WAR's library directory (`WEB-INF/lib`). Default value is `false`.
+| `mp.openapi.extensions.scan.lib`  |  Configuration property that allows the server to scan packaged JAR files inside the WAR's library directory (`WEB-INF/lib`). If the WAR is inside EAR, the server scans also JAR files inside EAR's `lib/` directory. Default value is `false`.
```
Comment (Pandrex247) [inline enterprise/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc:86]: Wording
```suggestion
| `mp.openapi.extensions.scan.lib`  |  Configuration property that allows the server to scan packaged JAR files inside the WAR's library directory (`WEB-INF/lib`). If the WAR is inside EAR, the server also scans JAR files inside the EAR's `lib/` directory. Default value is `false`.
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc (hunk 2 above)
Comment (aubi) [reply enterprise/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc:67]: The old rendering is wrong in the old version in the same way.

Comment (aubi) [reply enterprise/docs/modules/ROOT/pages/Technical Documentation/MicroProfile/OpenAPI.adoc:67]: This one was tricky: https://docs.asciidoctor.org/asciidoc/latest/text/quotation-marks-and-apostrophes/#possessive-monospace

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**